Cargo.toml

```diff
@@ -21,18 +21,18 @@ include = ["Cargo.toml", "LICENSE", "README.md", "src/**", "tests/**", "examples
 
 [dependencies]
 indexmap = "2.0.2"
-priority-queue = "1.1.1"
+priority-queue = "2.0.2"
 thiserror = "1.0"
 rustc-hash = "1.1.0"
 serde = { version = "1.0", features = ["derive"], optional = true }
 log = "0.4.14" # for debug logs in tests
 
 [dev-dependencies]
-proptest = "0.10.1"
-ron = "0.6"
```

Comment:
Change log at https://github.com/ron-rs/ron/blob/master/CHANGELOG.md#060---2020-05-21 which all seems reasonable. Although we also could use a bug fix from prerelease of `0.9`, but I have a separate PR for that.